client/state/signup/steps/jpo-connect/action.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+const debug = require( 'debug' )( 'calypso:jetpack-jpo:actions' );
```

Comment:
Sorry I missed this before! We're doing our best to use `import`s and remove `require`s.

This is usually done as:

```js
/**
 * External dependencies
 */
import debugFactory from 'debug';

/**
 * Module constants
 */
const debug = debugFactory( 'calypso:jetpack-jpo:actions' );
```

I went ahead an pushed a commit with this change 🙂